stage0/src/lib.rs

```diff
@@ -302,3 +310,11 @@ fn phys_to_virt(address: PhysAddr) -> VirtAddr {
     // We use an identity mapping throughout.
     VirtAddr::new(address.as_u64())
 }
+
+/// Overwrites `measurement` with the SHA2-384 digest or `source`.
+fn populate_measurement(measurement: &mut [u8; 48], source: &[u8]) {
+    let mut digest = Sha384::default();
```

Comment:
The binary hashes that are generated during build time of the binaries (which will be used as reference values) are all SHA 256 hashes. Therefore, IIUC all hashes generated here as part of measurement must also be 256 bits length instead of 384.

Reply:
Yes, good point. I will change the measurement hashes to SHA2-256 for now. We can later change it to support SHA2-384 once the provenance generations and release process supports it.